Library/Homebrew/shims/super/cc

```diff
@@ -435,15 +436,15 @@ if __FILE__ == $PROGRAM_NAME
 
   ####################################################################### main
 
-  dirname, basename = File.split($0)
+  dirname, basename = File.split($PROGRAM_NAME)
```

Comment:
Might need `require "English"` for this.

Reply:
Yup 👍🏻